src/uu/seq/src/seq.rs

```diff
@@ -299,7 +301,11 @@ fn print_seq(
         match format {
             Some(f) => {
                 let s = format!("{}", value);
-                Memo::run_all(f, &[s]);
+                let result = Memo::run_all(f, &[s]);
+                if result.is_err() {
+                    show_error!("%{}: invalid conversion specification", result.unwrap_err());
+                    exit(1);
+                };
```

Comment:
Here, I also think the error should just be propagated with `?`. It's a bit more complicated than `printf` because you'll have to change the signature of the `print_seq` function, though. If it's too much for this PR, let me know and we'll just open an issue for it.

Reply:
I tinkered a bit to see if I could just add this change onto this commit, but I think it might be better for another issue - I'll go ahead and make it. 

I see that if I change the signatures to be `UResult`s, [the code will generalize a BrokenPipe `UIoError` into a `dyn UError` (so I can no longer check the error kind)](https://github.com/uutils/coreutils/pull/3112/files#diff-8ef9575cb5c2b35e15751308bf63d6dbf3ac217a383790f9355bc5e13ab2fdb9L141) , and think this causes 4 test cases to fail with `cargo t`.